docs/source/configuring_zipkin.rst

```diff
@@ -166,6 +166,21 @@ zipkin.always_emit_zipkin_headers
     about 300us on every non-traced request.
 
 
+zipkin.request_context
+~~~~~~~~~~~~~~~~~~~~~~
+    If it contains a valid request attribute, this specifies the stack
+    for storing the zipin attributes. If the name is invalid or the option
+    is missing, attributes will be stored in a thread local context.
+    The syntax is a path in dotted notation, e.g. 'request.context.zipkin'.
```

Comment:
request.context.zipkin is misleading, as it suggests you might have to include the request object in the path.